public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/__snapshots__/reducer.test.tsx.snap

```diff
@@ -130,16 +130,16 @@ exports[`Query and expressions reducer should add reduce expression if there is
         },
         "expression": "A",
         "reducer": "last",
-        "refId": "B",
+        "refId": "reducer",
```

Comment:
Why `refId` is `reducer`?

Reply:
It's a random refid for the new reduce expression.